joomla/joomla-cms
# [4.0] Correcting broken Overrides Manager

### Summary of Changes
The filters are not displayed in 4.0, preventing a normal use of the Manager
This PR corrects the Overrrides Manager to get the same display as in 3.x

### Testing Instructions
Before patch. No language client filters. Filter Options also broken.
Patch then make sure that 2 files are deleted:

1. `administrator/components/com_languages/Field/Languageclient.php`
it is replaced in the patch by
`administrator/components/com_languages/Field/LanguageclientField.php`

2. `administrator/components/com_languages/layouts/joomla/searchtools/default/bar.php` has to be DELETED.
We now use the new file
`administrator/components/com_languages/layouts/joomla/searchtools/default.php`

EDIT: for those using patchtester, it will work ok if bar.php is not deleted.


### Before patch
No filters to choose from in order to create overrides.
Filter Options as well as column ordering and limit box are broken.

### After patch
Filters are back.
<img width="1284" alt="Screen Shot 2019-04-11 at 07 27 38" src="https://user-images.githubusercontent.com/869724/55932791-60ddab80-5c2b-11e9-83ad-81c3093a0e22.png">

<img width="1275" alt="Screen Shot 2019-04-10 at 17 32 19" src="https://user-images.githubusercontent.com/869724/55932833-7d79e380-5c2b-11e9-89cb-18c4768fca08.png">

<img width="488" alt="Screen Shot 2019-04-11 at 07 34 45" src="https://user-images.githubusercontent.com/869724/55933066-4f48d380-5c2c-11e9-9f63-3f8cb84a78ac.png">


@wilsonge @alikon 

Note: this patch does not prevent, for the Ajax specialists, to improve the whole stuff.
It just corrects the functionality as it works in 3.x
See discussion https://github.com/joomla/joomla-cms/pull/22107 , speciffically posts by @coolcat-creations
